### PR TITLE
WIP: Refactor Python code

### PR DESCRIPTION
**Pull request by kdjohnson**

### CODE REVIEW

**Comment by joshherkness** [comment]
Mmmmmm, this looks nice...

File: server/models.py [hunk 1]
```diff
@@ -66,16 +68,19 @@ def verify_auth_token(token):
...
     """
-    x = IntField(required=True, choices=range(1,max_size + 1))
-    y = IntField(required=True, choices=range(1,max_size + 1))
-    z = IntField(required=True, choices=range(1,max_size + 1))
+    x = IntField(required=True, choices=range(1, max_size + 1))
+    y = IntField(required=True, choices=range(1, max_size + 1))
+    z = IntField(required=True, choices=range(1, max_size + 1))
+
 
 class RGBColor(EmbeddedDocument):
```
Comment (joshherkness) [inline server/models.py]: This model can be removed, as we are using hex values for color now.

**Comment by joshherkness** [comment]
While you're at this, can you make all of the endpoints strip the user (if required) from the token instead of the email? Like, I don't want to pass email along with the `find_map` endpoint. I would rather pass my token along with the request, and have the endpoint determine who I am from my token, instead of the email.

**Comment by kdjohnson** [comment]
Yea I'll look into doing that.

**Review by AnthonyHewins — APPROVED** [review]

**Pull request merged**